storage/innobase/row/row0mysql.cc

```diff
@@ -2056,6 +2129,8 @@ row_update_for_mysql(row_prebuilt_t* prebuilt)
 		node->cascade_upd_nodes = cascade_upd_nodes;
 		cascade_upd_nodes->pop_front();
 		thr->fk_cascade_depth++;
+		vers_set_fields = node->table->versioned() &&
+				  (node->is_delete || node->versioned);
```

Comment:
In InnoDB, lines are split before operators such as `&&`, and the continuation lines are indented by one TAB when there is no unbalanced left parenthesis.